preamble.tex

```diff
@@ -159,8 +159,21 @@
 \preto{\appendix}{\cleardoublepage}
 
 \usepackage[style=american]{csquotes}  % Language sensitive quotation facilities
-\usepackage[style=authoryear-comp,backref=true,refsection=chapter,
-backend=biber]{biblatex}
+%\usepackage[style=authoryear-comp,backref=true,refsection=chapter,
+%backend=biber]{biblatex}
+
+% this might insert blank pages here and there but now compilation works correctly with biber
+\usepackage[
+	backend=biber,
+	sortlocale=us_EN,
+	natbib=true,
```

Comment:
Natbib option doesn't affect the authoryear-comp style AFAIR, therefore this settings is redundant.